src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java

```diff
@@ -381,20 +386,17 @@ public void readUIAction(EntityPlayerMP player, int id, PacketBuffer buf) {
     public void onLeftClick(EntityPlayer player, EnumFacing facing, CuboidRayTraceResult hitResult) {
         if (this.getWorld().isRemote) return;
         Pair<Double, Double> clickCoords = this.checkLookingAt(player);
-        if (this.guiContainerCache != null && guiContainerCache.modularUI != null) {
-            int width = guiContainerCache.modularUI.getWidth();
-            int height = guiContainerCache.modularUI.getHeight();
-            double scale = 1.0 / Math.max(width, height);
-            int mouseX = (int) ((clickCoords.getLeft() / scale));
-            int mouseY = (int) ((clickCoords.getRight() / scale));
-            if (0 <= mouseX && mouseX <= width && 0 <= mouseY && mouseY <= height) {
-                this.writeCustomData(MOUSE_POSITION, buf -> {
-                    buf.writeVarInt(mouseX);
-                    buf.writeVarInt(mouseY);
-                });
-            }
+        int width = 178; // These should always be correct.
+        int height = 230;
+        double scale = 1.0 / Math.max(width, height);
```

Comment:
Something that doesn't matter, the scale also be confirmed.

Reply:
Well, this should be fine, since it's using less magic numbers here.